packages/interaction/src/InteractionManager.ts

```diff
@@ -521,212 +521,6 @@ export class InteractionManager extends EventEmitter
          * @param {PIXI.InteractionEvent} event - Interaction event
          */
 
-        /**
```

Comment:
Why are you removing these event docs? Maybe we need to add something to this that's mention it's from `@pixi/interaction`? We're going to need to have duplicate events.

Reply:
Hmm, I can leave them there. It will be a bit weird for each event to be listed twice.

Reply:
That's why I think we need to list the packages in both because they have different params.

Reply:
Done!